CONTRIBUTING.md

```diff
@@ -353,6 +353,16 @@ req_set_exception_breakpoints([])
 
 to clear all exception breakpoints.
 
+- res_set_function_breakpoints(breakpoints)
+
+Sends request to rdbg to set function breakpoints. e.g.
+
+```rb
+req_set_function_breakpoints([{ name: "Foo#bar", condition: "a == 1" }])
```

Comment:
For me, I'd prefer to use `req_add_function_breakpoint(breakpoint)` to match with `req_add_breakpoint`.

Reply:
Because it can also remove breakpoints, I think it's better to use `set`. I prefer renaming other helpers instead, wdyt?